src/config/builder.rs

```diff
@@ -473,7 +473,7 @@ mod tests {
     /// the `ConfigBuilder` has changed what it serializes, or the implementation of `serde_json` has changed.
     /// If this test fails, we should ideally be able to fix so that the original hash passes!
     fn version_hash_match() {
-        let expected_hash = "6c98bea9d9e2f3133e2d39ba04592d17f96340a9bc4c8d697b09f5af388a76bd";
+        let expected_hash = "9bc035fa506e00403e479451dc8bf1e566a44f7ac83ccde349bd6c5a9bd18384";
```

Comment:
As we have now added a new field to `api` which is a field of the`ConfigBuilder` struct, the serialization of the `ConfigBuilder` results in a different hash. I have not yet found a way to remove the added `graphql` field from the serialization without composing a replica without the `graphql` field. Input would be greatly appreciated!

Reply:
Ah, yes, I saw this too. You'll need to add a `#[serde(skip_serializing_if = "…")]` notation to the new field.